apps/platform-app/src/app/[locale]/(profile)/(components)/CVuploader/CvUploader.tsx

```diff
@@ -1,12 +1,11 @@
-import { CVuploaderForm } from "./CvUploaderForm"
 import styles from './CvUploader.module.scss'
+import { CVuploaderForm } from './CvUploaderForm'
```

Comment:
**nitpick**: Import statements should be alphabetically ordered for consistency.